### DISCUSSION TITLE
Inline maths rendering bug

### DISCUSSION THREAD

**Discussion by gititGoro** [community/community#30606] [Other Feature Feedback, Questions, & Ideas] [answered]
As of today (August 24 2022), multiple inline maths expressions don't render. Observe, the first instance of $Ψ$ renders correctly.

If I add a newline, then from then on, inline math rendering fails. 

This gist demonstrates that in action https://gist.github.com/gititGoro/b3ca31572976468991f46a71ccbba376

Otherwise, here is a demonstration: The following term will not render $D_F=3*2$

Note this bug didn't exist yesterday.

**Comment by ElectronicsArchiver** [comment]
Yep, my whole project is unusable because of that..

Just look at this .. (;´༎ຶД༎ຶ`)
https://github.com/OpenlyEducated/Mathematics/blob/main/Areas/Sets/Roaster%20Notation.md

**Comment by gititGoro** [comment]
I've noticed that one important step to reproducing this bug is to have more than 1 newline and some additional text after the initial inline expression. If additional inline maths expressions occur in the same paragraph as the original, they still render.

**Reply by ElectronicsArchiver** [reply]
It seems they fixed the math code block and with that broke the inline version..

**Reply by Nicholaswogan** [reply]
Do you know the rough timescale for fixing this kind of stuff is? 1 week? 1 month?

**Reply by JensMunkHansen** [reply]
I reported the bug 3 days ago. Last week, it was working okay. I really hope somebody will look into this. All inline math is broken after the first non-inline math.

**Reply by ElectronicsArchiver** [reply]
Which of GitHub repos even contains the Markdown - MathJax stuff?

**Reply by JensMunkHansen** [reply]
We use it at work for documentation of source code in markdown files residing next to the source. It is very convenient

**Reply by ElectronicsArchiver** [reply]
@JensMunkHansen Sorry, I meant which of GitHubs own repositories contains their code and thus their introduced bug. But yes, it sure is useful~

**Comment by nschloe** [comment]
> Do you know the rough timescale for fixing this kind of stuff is? 1 week? 1 month?

They make some changes for math mode about once a month, but there are fundamental problems that were present on release day that haven't been fixed yet, leading me to believe they don't have a lot of devpower behind this operation. I wouldn't hold my breath.

Perhaps it's a good idea to make a test repo for yourself, containing the things that don't work, such that you can check it each it week or so. I've got that at https://github.com/nschloe/math-test.

**Comment by GuillaumeBiessy** [comment]
I'm having the same issue with none of my inline math working in this repository : 
https://github.com/GuillaumeBiessy/WH

Ten days ago, the inline math was working fine. Now, none of it is working.
Besides there are some weird errors in display math as well with some subscripts not being displayed at subscripts. See the second display equation in the above link.

**Comment by ElectronicsArchiver** [comment, marked as answer]
It seems to work again?

https://github.com/OpenlyEducated/Mathematics/blob/main/Areas/Sets/Roaster%20Notation.md

1. Broken but 3 blocks weren't anymore
2. Went into the editor to check, preview wasn't broken anymore
3. Went back without committing, still broken
4. When into the editor, removed a ( random ) newline ( any change to commit )
Fixed..

Weird, I thought I didn't properly clear my cache since other 
files were already working again, though in another repository 
they weren't fixed yet so maybe I'm not crazy after all..

**Comment by gititGoro** [comment]
I'm also experiencing this as fixed.

One slight detail. Not sure if this was there before but just be careful with parentheses. In VSCode, 
($X<3$) renders but not in Github as can be seen. My workaround is to add a space between like this  ( $X<3$ )

**Reply by yasirroni** [reply]
I support you to open again this issue (or ipen new issue regarding `($`. For now, I submitted feedback to GitHub here: https://support.github.com/contact/bug-report

**Reply by nschloe** [reply]
Yeah, you can't put anything around the dollar delimiters for now. From what I hear, they're working on revamping math support.

**Comment by Shungy** [comment]
Inline math here doesn't render as well: https://github.com/pangolindex/exchange-contracts/tree/main/contracts/staking-positions#sar-proof
They were rendering fine a week or so ago.

**Reply by gititGoro** [reply]
If I snag that paragraph and render it on its own, it works. I suspect something further up in the document is breaking it such as an errant open parenthesis.
Here's a gist to prove it: https://gist.github.com/gititGoro/ddae2cce2280d717baa4ae2a7b5f2561


Interesting project, by the way.  I'm also writing a yield farm based on some unique mechanics and need to publish a paper explaining some of the economics :) 

I like your use of total staked time as a deflator of rewards. It delinearizes the incentives to wait which is a refreshing take on simple lockup mechanisms like ve tokens.

**Reply by Shungy** [reply]
You were mostly right. Strangely, it was due to `textit` in a previous math block in the document. Fixed it by changing to `text`: https://github.com/pangolindex/exchange-contracts/commit/23b4e75cd341f91b1bf0af34932bbba9c21f0cef

>Interesting project, by the way.

Thanks :D
